test/followers_institution_test.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""Institution follower handler test."""
+
+import unittest
+from test_base import TestBase
+from models.user import User
+from models.institution import Institution
+from handlers.institution_followers_handler import InstitutionFollowersHandler
+
+
+class InstitutionFollowersHandlerTest(TestBase):
+    """Test the post_handler class."""
+
+    @classmethod
+    def setUp(cls):
+        """Provide the base for the tests."""
+        cls.test = cls.testbed.Testbed()
+        cls.test.activate()
+        cls.policy = cls.datastore.PseudoRandomHRConsistencyPolicy(
+            probability=1)
+        cls.test.init_datastore_v3_stub(consistency_policy=cls.policy)
+        cls.test.init_memcache_stub()
+        cls.ndb.get_context().set_cache_policy(False)
+        app = cls.webapp2.WSGIApplication(
+            [("/api/institution/(.*)/followers",
+                InstitutionFollowersHandler)
+             ], debug=True)
+        cls.testapp = cls.webtest.TestApp(app)
+        initModels(cls)
+
+    def mayza_follow_splab(self):
+        """Test with institution SPLAB and user MAYZA."""
+        # Pretend an authentication
+        self.os.environ['REMOTE_USER'] = 'mayzabeel@gmail.com'
+        self.os.environ['USER_EMAIL'] = 'mayzabeel@gmail.com'
+        # Verified objects, are empty
+        self.assertTrue(len(self.splab.followers) == 0)
```

Comment:
colocar mensagem em todos os asserts.
Exemplo: self.assertTrue(len(self.splab.followers) == 0, "The number of followers expected was 0")